### PR TITLE
[11.0] [MIG] account_reconcile_payment_order

### PR DESCRIPTION
**Pull request by pedrobaeza**
cc @Tecnativa

### CODE REVIEW

**Review by Tardo — APPROVED** [review]
LGTM :+1:

**Review by rafaelbn — APPROVED** [review]
:+1:

File: account_reconcile_payment_order/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Reconcile payment orders",
+    "version": "11.0.1.0.0",
+    "author": "Therp BV,"
+              "Tecnativa,"
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
```
Comment (rafaelbn) [inline account_reconcile_payment_order/__manifest__.py:11]: Add "website": https://github.com/OCA/account-reconcile/

**Review by rafaelbn — CHANGES_REQUESTED** [review]

File: account_reconcile_payment_order/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+* Holger Brunn
+* `Tecnativa <https://www.tecnativa.com>`__:
+
+  * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```
Comment (rafaelbn) [inline account_reconcile_payment_order/readme/CONTRIBUTORS.rst]: Review this email address

**Review by rafaelbn — CHANGES_REQUESTED** [review]

**Pull request merged**